.github/workflows/build_linux_pytorch_wheels.yml

```diff
@@ -73,9 +75,14 @@ jobs:
           git config --global user.name "therockbot"
           git config --global user.email "therockbot@amd.com"
 
+      - name: Select Python version
+        run: |
+          python build_tools/github_actions/python_to_cp_version.py \
+            --python-version ${{ inputs.python_version }}
+
       - name: Add selected Python version to PATH
```

Comment:
I could see merging these steps into one since no other steps depend on their intermediate outputs. Up to you.

Reply:
I tried that initially but the environment variable wasn't usable within the step and we would need to capture the output
```bash
cp_version=`python build_tools/github_actions/python_to_cp_version.py --python-version ${{ inputs.python_version }}`
```
instead of writing to the env file. However, I think it is desirable to see the output which makes it also easier to see failures.

Reply:
You could also set an environment variable and not use GITHUB_ENV, as long as the variable is only used within the current step.

How you have it now is fine though.